### PR TITLE
ffh.gw-statuspage: introduce role

### PR DESCRIPTION
**Pull request by AiyionPrime**
a gluon like statuspage in ff-blue, written in rust

### CODE REVIEW

**Comment by AiyionPrime** [comment]
The role has been tested and is currently deployed on [sn07](http://sn07.s.ffh.zone/cgi-bin/status)

**Comment by AiyionPrime** [comment]
The only thing left for an alphaphase on all supernodes would be to get rid of the dependency on the directory `aiyion_test`, which will take place in the next version of gw-statuspage, but does not block this.

File: roles/ffh.gw-statuspage/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    repo=https://github.com/freifunkh/site.git
+    dest=/var/cache/gw-statuspage/site
+    accept_hostkey=true
+  register: site
+
+- name: Add aiyions package-repo into sources list
+  ansible.builtin.apt_repository:
+    repo: deb http://aiyionpri.me/repo/ ./
+    state: present
+    filename: aiyionprime
```
Comment (1977er) [inline roles/ffh.gw-statuspage/tasks/main.yml:40]: Sounds a bit generic. Will this deb be the only file you are serving on that repo?

**Review by 1977er — APPROVED** [review]
Apart from my comment looks good.

File: roles/ffh.gw-statuspage/tasks/main.yml (hunk 1 above)
Comment (AiyionPrime) [reply roles/ffh.gw-statuspage/tasks/main.yml:48]: Unlikely, but for now only each new version of gw-statuspage will be there.

**Comment by AiyionPrime** [comment]
Just saw, the master branch has the "unconfigured" commit as well.
No need to use nightly for now.

I just released 0.1.1-1 of gw-statuspage, which does not need the `aiyion_test` directory anymore.
Please do not delete it from sn07 though.

I'd test this on a different sn and if that works out, I'll merge this. Any objections?

**Pull request merged**